ui/appui-layout-react/src/test/widget/FloatingWidget.test.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`FloatingWidget should render 1`] = `
 <div
-  class="nz-widget-widget nz-widget-floatingWidget nz-hidden"
+  class="nz-widget-widget nz-widget-floatingWidget"
```

Comment:
This changes the default behavior of  `FloatingWidget` auto-hide feature: it is not hidden by default anymore, unless `hideWithUiWhenFloating` is set. Should we consider setting `hideWithUiWhenFloating` to `true` as a default value in `WidgetDef` or we are fine with this change?

Reply:
I think because it was broken for so long, we should let people opt in.